packages/hooks/package.json

```diff
@@ -17,16 +17,64 @@
     "type": "git",
     "url": "git+https://github.com/Developer-DAO/web3-ui/"
   },
-  "main": "index.js",
   "scripts": {
-    "types": "index.d.ts",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "yarn clean; rollup -c",
+    "clean": "rimraf dist",
+    "format": "prettier --write \"src/**/*.{ts,tsx,json,js,jsx}\"",
+    "format:check": "prettier --list-different \"src/**/*.{ts,tsx,json,js,jsx}\"",
+    "test": "jest --maxWorkers=2",
+    "test:watch": "yarn test --watch",
+    "test:coverage": "jest --coverage --colors --maxWorkers=2",
+    "hygen": "hygen",
+    "component:new": "hygen component with-prompt",
+    "pre-commit-hook": "yarn lint-staged",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook",
+    "dev": "yarn storybook"
   },
-  "keywords": [],
+  "main": "dist/index.js",
+  "module": "dist/index.esm.js",
+  "types": "dist/index.d.ts",
   "dependencies": {
-    "react": "^17.0.2"
+    "@walletconnect/web3-provider": "^1.6.6",
+    "ethers": "^5.5.1",
+    "web3modal": "^1.9.4"
   },
   "peerDependencies": {
-    "react": ">=16.8.6"
+    "react": "*",
```

Comment:
At a minimum this needs to be `16.8.0` because that's when React added hooks support.